cassandra-scheduler/src/main/java/com/mesosphere/dcos/cassandra/scheduler/resources/TasksResource.java

```diff
@@ -141,8 +145,9 @@ public void replace(@QueryParam("node") final String name)
     @GET
     @Path("connect")
```

Comment:
Please see https://github.com/mesosphere/dcos-cassandra-service/pull/233 for deprecation notes.